packs/equipment-effects/effect-fury-cocktail-greater.json

```diff
@@ -0,0 +1,241 @@
+{
+    "_id": "6lzmzeWRo6mv6g2C",
+    "img": "systems/pf2e/icons/equipment/alchemical-items/alchemical-elixirs/fury-cocktail.webp",
+    "name": "Effect: Fury Cocktail (Greater)",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Fury Cocktail (Greater)]</p>\n<p>Implemented Effects:</p>\n<ul>\n<li>+4 item bonus to melee attack rolls</li>\n<li>-1 penalty to AC</li>\n<li>-2 penalty to Reflex saves</li>\n<li>Animalistic: choice of Jaw or Claw strike</li>\n<li>Double: resistance 5 to physical damage</li>\n<li>Mournful: resistance 10 to void or vitality depending on negative healing</li>\n<li>Skeptical: +3 item bonus to saves against magic</li>\n<li>Titanic: size change to Large, Clumsy 1, and change of reach (+5 for Medium and Small, +10 for Tiny)</li>\n<li>Wyrmhide: resistance 10 to acid, cold, electricity, fire, and poison.</li>\n</ul>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "hours",
+            "value": 1
+        },
+        "level": {
+            "value": 1
+        },
+        "publication": {
+            "license": "OGL",
+            "remaster": false,
+            "title": "Pathfinder Treasure Vault"
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "selector": "reflex",
+                "value": -2
+            },
+            {
+                "choices": [
+                    {
+                        "label": "Animalistic",
```

Comment:
We typically want to localize labels. I'll leave it to @TikaelSol to decide on whether these should be separate effects. I can see the appeal in having 3 rather than 18 though.